resources/metadata/golangBuild.yaml

```diff
@@ -145,6 +149,27 @@ spec:
           - STEPS
           - STAGES
           - PARAMETERS
+      - name: privateModules
+        type: "string"
```

Comment:
Just that I get it right. Instead of a single glob pattern, you want to support a set of glob patterns? Like:

```yaml
  golangBuild:
    privateModules:
      - git.company.local/my/module1
      - git.company.local/my/module2"
      - git.company.local/3rdparty/*
      - "git.company.local/my/module1,git.company.local/my/module2"
      - *.othercompany.local
```

Personally I don't see a benfit, but if you prefer to have it like this I don't mind.

Reply:
To sum up the discussions we had offline. We'll stick with the privateModules parameter as it is, because this is the way how go developers are used to configure their local environments.